.travis.yml

```diff
@@ -2,9 +2,15 @@ language: php
 php:
   - "5.4"
   - "5.5"
+  - "5.6"
+  - "nightly"
   - hhvm
 
 before_script:
   - composer install --dev
 
 script: ./vendor/bin/phpunit --coverage-text
+
+matrix:
+     allow_failures:
+         - php: hhvm  
```

Comment:
I'm thinking of a way of solving this at the moment... for some reason it's just DateTimeZone and DateInterval classes that are failing.